File: lua/refactoring/tests/debug/print_var/go/simple-function/print_var.expected.go
package main

import "fmt"

func simple_function(a int) {
	var test int = 1
	test_other := 1
fmt.Println(fmt.Sprintf("simple_function test_other: %v", test_other))
	for idx := test - 1; idx < test_other; idx++ {
		fmt.Println(idx, a)
	}
}

File: lua/refactoring/tests/debug/print_var/lua/indexed-expression/print_var.expected.lua
-- stylua: ignore start
local function simple_function()
    local test = {
        test_other = 11,
    }
    print(test.test_other)
    -- __AUTO_GENERATED_PRINT_VAR_START__
    print([==[simple_function test.test_other:]==], vim.inspect(test.test_other)) -- __AUTO_GENERATED_PRINT_VAR_END__
end
